sdcm/cluster_cloud.py

```diff
@@ -315,6 +316,30 @@ def raft(self):
             is_ready=lambda: True)
 
 
+class CloudVSNode(CloudNode):
```

Comment:
@dimakr can you raise a task for doing a manager node, I'm quite sure @mikliapko would need for manager related tests that would need to be backported from siren-tests